wix.json

```diff
@@ -0,0 +1,27 @@
+{
+  "product": "MongoDB CLI",
+  "company": "MongoDB, Inc.",
+  "license": "LICENSE",
+  "upgrade-code": "a995cda5-9731-4158-910c-c8d15371dbda",
+  "files": {
+    "guid": "c7b0df57-c3c1-4591-becd-f0f73a912eb8",
+    "items": [
+      "mongocli.exe"
+    ]
+  },
+  "env": {
+    "guid": "94faac3d-4478-431c-8497-fba55dcfb249",
```

Comment:
[q] Does this need to change for each version, so should be generated? My memories of msi is each installer has a new guide and the upgrade-code is how you track that it's the same product.

Reply:
according to go-msi this should be done just once `go-msi set-guid` so hopefully this won't change again(?)

Reply:
Cool, fingers crossed. I remember in the past changing the product guid each time, but needing to keep the same upgrade guid to link releases. This will come out in the wash if there's a problem.